packages/common/src/core/HubItemEntity.ts

```diff
@@ -236,10 +236,19 @@ export abstract class HubItemEntity<T extends IHubItemEntity>
    * Returns the followers group
    */
   async getFollowersGroup(): Promise<IGroup> {
-    return getGroup(
-      this.entity.followersGroupId,
-      this.context.userRequestOptions
-    );
+    let group = {} as IGroup;
```

Comment:
I'm not super wild about this approach b/c consumers need to check more than just "is the returned object defined"... was there a reason to not let `getGroup` throw?